app/models/transformation_mapping.rb

```diff
@@ -94,6 +95,11 @@ def validate_vm(vm, quick = true)
     # a valid vm must find all resources in the mapping and has never been migrated
     invalid_list = []
 
+    if vm.ext_management_system.nil?
```

Comment:
remove Ln 98 - 102.
Add following to https://github.com/ManageIQ/manageiq/blob/master/app/models/vm.rb#L125
```
return TransformationMapping::VM_INACTIVE unless active?
```

Reply:
With this change, the return reason I get is undefined and not `VM_INACTIVE` because the return happens from here - https://github.com/ManageIQ/manageiq/blob/master/app/models/transformation_mapping.rb#L99

and not from `validate_v2v_migration`

Reply:
I think the `active?` check needs to happen before we call this block 

```rb
unless valid_cluster?(vm)
  invalid_list << "cluster: %{name}" % {:name => vm.ems_cluster.name}
  return no_mapping_msg(invalid_list) if quick
end
```

Reply:
@bzwei I had to go with the implementation in c2b8ec7, for the reason mentioned above.